docs/sources/datasources/elasticsearch/configure-elasticsearch-data-source.md

```diff
@@ -0,0 +1,153 @@
+---
+aliases:
+  - ../data-sources/elasticsearch/
+  - ../features/datasources/elasticsearch/
+description: Guide for configuring the Elasticsearch data source in Grafana
+keywords:
+  - grafana
+  - elasticsearch
+  - guide
+  - data source
+labels:
+  products:
+    - cloud
+    - enterprise
+    - oss
+menuTitle: Configure Elasticsearch
+title: Configure the Elasticsearch data source
+weight: 200
+---
+
+# Configure the Elasticsearch data source
+
+Grafana ships with built-in support for Elasticsearch.
+You can make many types of queries to visualize logs or metrics stored in Elasticsearch, and annotate graphs with log events stored in Elasticsearch.
+
+For general documentation on querying data sources in Grafana, see [Query and transform data]({{< relref "../../panels-visualizations/query-transform-data" >}}).
+
+For instructions on how to add a data source to Grafana, refer to the [administration documentation]({{< relref "../../administration/data-source-management/" >}}).
+Only users with the organization administrator role can add data sources.
+Administrators can also [configure the data source via YAML]({{< relref "#provision-the-data-source" >}}) with Grafana's provisioning system.
+
+## Configure the data source
+
+To add the Elasticsearch data source, complete the following steps:
+
+1. Click **Connections** in the left-side menu.
+1. Under **Connections**, click **Add new connection**.
+1. Enter `Elasticsearch` in the search bar.
+1. Select **Elasticsearch data source**.
+1. Click **Create a Elasticsearch data source** in the upper right.
+
+You will be taken to the **Settings** tab where you will set up your Elasticsearch configuration.
+
+## Configuration options
+
+The following is a list of configuration options for Elasticsearch.
+
+The first option to configure is the name of your connection:
```

Comment:
I'm not sure this is necessary? It's not wrong, it's just that the user can see that **Name** is the first config option so not sure if it's valuable to point that fact out. A pure nit on my part :)